uint16 beepLength  # length of beep on the hardware
uint8 numberOfBeeps  # length of beep on the hardware
---
